uint64 uid
uint8 object_class
hsr_planner/CellMessage center_cell
float32 center_wx
float32 center_wy
hsr_planner/CellMessage[] cell_vector